elife/config/etc-cron.daily-logrotate

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+# Clean non existent log file entries from status file
+cd /var/lib/logrotate
+test -e status || touch status
+head -1 status > status.clean
+sed 's/"//g' status | while read logfile date
+do
+    [ -e "$logfile" ] && echo "\"$logfile\" $date"
+done >> status.clean
+mv status.clean status
+
+test -x /usr/sbin/logrotate || exit 0
+capture_file=/tmp/daily-logrotate.out
+/usr/sbin/logrotate /etc/logrotate.conf --verbose &> "$capture_file" || {
```

Comment:
`&>` should redirect both stdout and stderr, but I've usually seen it without spaces between operator and file name.

Reply:
like this? ` --verbose &>"$capture_file" ` 

I can't say I've seen it consistently done one way or the other. 

I think cron may be futzing with the stdout/stderr streams anyway, I need to do some more investigation

Reply:
I confirm it works in all expected cases :+1: